schema/event_mapping_entities.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$udmi_version": "1.4.0",
+  "title": "Mapping Event Entities",
+  "description": "Mapper output describing a group of devices by their guids",
+  "type": "object",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "additionalProperties": false,
+  "existingJavaType": "java.util.HashMap<String, MappingEventEntity>",
+  "patternProperties": {
+    "": {
```

Comment:
what are these keys?  There should be some format to them... "anything goes" isn't quite what we want.

Reply:
you can just use that one you see in a few places -- I just want something that's reasonably accurate in there!